src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        printGreeting();
+
+        try {
+            loadTasks();
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        Scanner in = new Scanner(System.in);
```

Comment:
Maybe consider using a name just like "scanner" for the Scanner object so that it explains a little bit more of the purpose of the variable?